tutorial/westeros/westeros_firm_capacity.ipynb

```diff
@@ -1,5 +1,16 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Westeros Tutorial Part IVa. - Introducing firm capacity\n",
```

Comment:
The term "firm capacity" is quite specific, so I would use it in quotes when first used. Also, I would make it clear that the description below is the explanation for that term.